src/Uno.UI/UI/Xaml/Media/RadialGradientBrush.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Windows.Foundation;
+using Windows.UI.Composition;
+using Windows.UI.Xaml.Markup;
+
+namespace Windows.UI.Xaml.Media
+{
+	public sealed partial class RadialGradientBrush : GradientBrush
+	{
+		public static readonly DependencyProperty CenterProperty = DependencyProperty.Register(
+			"Center", typeof(Point), typeof(RadialGradientBrush), new PropertyMetadata(new Point(0.5d, 0.5d)));
+
+		public Point Center
+		{
+			get => (Point)GetValue(CenterProperty);
+			set => SetValue(CenterProperty, value);
+		}
+
+		public static readonly DependencyProperty RadiusXProperty = DependencyProperty.Register(
+			"RadiusX", typeof(double), typeof(RadialGradientBrush), new PropertyMetadata(1.0d));
+
+		public double RadiusX
+		{
+			get => (double)GetValue(RadiusXProperty);
+			set => SetValue(RadiusXProperty, value);
+		}
+
+		public static readonly DependencyProperty RadiusYProperty = DependencyProperty.Register(
+			"RadiusY", typeof(double), typeof(RadialGradientBrush), new PropertyMetadata(1.0d));
+
+		public double RadiusY
+		{
+			get => (double)GetValue(RadiusYProperty);
+			set => SetValue(RadiusYProperty, value);
+		}
+
+		public static readonly DependencyProperty GradientOriginProperty = DependencyProperty.Register(
+			"GradientOrigin", typeof(Point), typeof(RadialGradientBrush), new PropertyMetadata(default(Point)));
```

Comment:
```suggestion
			"GradientOrigin", typeof(Point), typeof(RadialGradientBrush), new PropertyMetadata(new Point(0.5, 0.5)));
```

https://github.com/microsoft/microsoft-ui-xaml/blob/ca95be6f5248eaa55ec523f2d6e7b50f6e0d53e7/dev/RadialGradientBrush/RadialGradientBrush.idl#L24